packages/tempus/src/Tempus_StepperImplicit_decl.hpp

```diff
@@ -248,14 +249,14 @@ class StepperImplicit : virtual public Tempus::Stepper<Scalar>
 
     /// Set solver.
     virtual void setSolver(
-      Teuchos::RCP<Thyra::NonlinearSolverBase<Scalar> > solver);
+      Teuchos::RCP<Thyra::NonlinearSolverBase<Scalar> > solver) override;
```

Comment:
const?

Reply:
Probably not.  We probably want to allow the user to modify it.